src/content/structured/components/date-picker/guidance.mdx

```diff
@@ -144,13 +158,28 @@ Set `open-at-date` to specify the date in view when the date picker is opened. I
 
 The [date input](/components/date-input) and the date picker update according to any input made. For example, if 10/10/2100 is entered in the [date input](/components/date-input), the date picker will open to display the selection.
 
+Use the month and year buttons to quickly navigate between larger spans of time to find the intended date.
+
+<DoDontCaution
+  imageSrc={[Fig5DatePickerLight, Fig5DatePickerDark]}
+  imageAlt="Three date pickers displaying the differing content views of day, month and year."
+  caption="Use the date picker month/year navigation buttons to display individual views of that type of selection."
```

Comment:
I think we've generally avoided punctuation like this to make things more readable
```suggestion
  caption="Use the date picker month or year navigation buttons to display individual views of that type of selection."
```